lib/runners/julia.js

```diff
@@ -12,7 +12,7 @@ module.exports.run = function run(opts, cb) {
       if (opts.setup) codeWriteSync('julia', opts.setup, juliaCodeDir);
       runCode({
         name: 'julia',
-        args: ['-P', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
+        args: ['', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
```

Comment:
Maybe that first empty string is causing the issue.

Why not `julia -q file` with `cwd` set to `juliaCodeDir` instead?
Is there a reason to include test framework in `solutionOnly`?

```js
runCode({
  name: 'julia',
  args: ['-q', opts.solution],
  options: {
    cwd: juliaCodeDir,
  }
});
```